tests/x11/firefox.pm

```diff
@@ -23,9 +23,10 @@ sub run() {
 
     $self->start_firefox;
     wait_still_screen;
-    send_key_until_needlematch('firefox-top-bar-highlighted', 'alt', 4, 10);
-    send_key('h');
-    wait_still_screen;
+    send_key('alt', 5);
+    send_key('alt-v', 5);
+    assert_screen('firefox-top-bar-highlighted');
+    send_key('alt-h', 5);
```

Comment:
I think reserve the 'wait_still_screen;' will be helpful for timing issue.

Reply:
send_key
send_key($key [, wait_screen_change => $wait_screen_change]);
It same to set 5s at send_key.